diff --git a/pkg/clientset/versioned/fake/register.go b/pkg/clientset/versioned/fake/register.go
index ddabbb93..1faaf8cb 100644
--- a/pkg/clientset/versioned/fake/register.go
+++ b/pkg/clientset/versioned/fake/register.go
@@ -36,7 +36,6 @@ import (
 
 var scheme = runtime.NewScheme()
 var codecs = serializer.NewCodecFactory(scheme)
-var parameterCodec = runtime.NewParameterCodec(scheme) // noolint
 var localSchemeBuilder = runtime.SchemeBuilder{
 	applicationv1alpha1.AddToScheme,
 	backupv1alpha1.AddToScheme,
